README.md

```diff
@@ -16,3 +16,15 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+In order to verify said claim, we must first look at the best case runtime. According to the slides/video, the best run time
+for a comparison algorithm would be $O(nlogn)$, so, this run time would be incorrect. Looking further,
+the binary tree for each decision including leaves of true or false must have at least a height of $n!$, so the given time 
+would be incorrect. So, therefore, while the best case we may expect $O(nlogn)$, this proves the given run time
+for X to be incorrect. If I were to load in 2, 3, 4, 1, the tree would need a height of 4!, and would need to test each decision
+to make the right choice on whether it is sorted or not. The tree is bases on the decisions, not the numbers loaded in. Each leaf
+would essentially include a decision, looking at whether a[0] > a[1], and continues to see if this is true or false. 
```

Comment:
Still don't understand where you're going with this. We used the tree for the theoretical analysis; when actually sorting there is no tree.